tgui/packages/tgui/interfaces/NovaCharacterDirectory.jsx

```diff
@@ -0,0 +1,253 @@
+import { useState } from 'react';
+
```

Comment:
```suggestion
// THIS IS A NOVA SECTOR UI FILE
import { useState } from 'react';

```

Also this is a requirement

Reply:
Denk u. All of these suggestions are committed. Resolving converLAMEtion